core/src/data/row.rs

```diff
@@ -99,16 +107,23 @@ impl Row {
 
                 match (value, column_def.get_default(), nullable) {
                     (Some(&expr), _, _) | (None, Some(expr), _) => {
-                        evaluate_stateless(None, expr)?.try_into_value(data_type, *nullable)
+                        let value =
+                            evaluate_stateless(None, expr)?.try_into_value(data_type, *nullable)?;
+
+                        Ok(value)
```

Comment:
Hmm.. I think we can leave it as it is.

Reply:
Oh it looks that this is unnecessary change I've made while.. may be during doing some tests. original one looks enough I'll change it to the prev